lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -76,18 +78,23 @@ export class StartProcessCloudComponent implements OnChanges, OnInit {
     processForm: FormGroup;
     processPayloadCloud = new ProcessPayloadCloud();
     filteredProcesses: ProcessDefinitionCloud[] = [];
+    maxProcessInstanceNameLength: number = this.MAX_LENGTH;
```

Comment:
/** Maximum length of the task name. */
    @Input()
    maxNameLength: number = StartTaskCloudComponent.MAX_NAME_LENGTH;

Reply:
fixed